modules/developer_manual/pages/general/backporting.adoc

```diff
@@ -56,6 +56,9 @@ limit allows for up to 60 requests per hour. Unauthenticated requests are
 associated with the originating IP address, and not the user making requests.
 For more information see https://developer.github.com/v3/#rate-limiting
 
+NOTE: In case of conflicts, the script ends and you have to solve them
```

Comment:
This sentence doesn't, quite, make sense. I suggest:

> In case of conflicts, the script exits. The merge conflicts will need to be resolved before manually continuing the backport.